Sphinx/Contributing/Writing Docs.rst

```diff
@@ -88,16 +91,16 @@ Classes
     follow the `Classes <https://google.github.io/styleguide/pyguide.html#384-classes>`_
     guidelines for content and format, except for the ``__init__`` method.
     The class *docstring* should also contain a description of the ``__init__``
-    methord and its arguments.
+    method and its arguments.
 
 
-Adding Documentation for a Module
-=================================
+Adding Documentation for Modules
+================================
```

Comment:
It would be easier to not use the alt header markdown...

`# Adding Documentation for Modules`

`## Module Documentation`

Reply:
Not sure what you mean?

Reply:
===== and ----- are alternatives to using # and ##, respectively. Or I'm missing something...

Reply:
No, reST is different than md for headers. See here http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections

Reply:
Ah...